src/components/buttons/secondary-button/secondary-button.js

```diff
@@ -141,9 +140,22 @@ SecondaryButton.propTypes = {
     );
   },
 
-  onClick: requiredIf(PropTypes.func, props => !props.linkTo),
+  onClick: requiredIf(PropTypes.func, props => {
+    return !props.linkTo && !props.to;
+  }),
   as: PropTypes.oneOfType([PropTypes.string, PropTypes.elementType]),
-  to: PropTypes.string,
+  to(props, propName, componentName, ...rest) {
+    if (props[propName] != null) {
+      if (!props.as) {
+        return new Error(oneLine`
+                Invalid prop "${propName}" supplied to "${componentName}".
+                "${propName}" does not have any effect when "as" is not defined`);
```

Comment:
This formatting looks a bit funny. Sorry.

Reply:
why do we even have prettier 🤔

Reply:
Cause it doesn't format template literals :D

Reply:
oh well